resources/istio/templates/sidecar-injector-configmap.yaml

```diff
@@ -18,9 +18,9 @@ data:
       - name: istio-init
         resources:
           limits:
-            memory: 96Mi
-          requests:
             memory: 32Mi
+          requests:
+            memory: 16Mi
```

Comment:
Here you set memory for init-container which I think is not that important. Resources for istio proxy are defined in [`resources/istio/values.yaml`](https://github.com/kyma-project/kyma/blob/master/resources/istio/values.yaml#L39). You can set them via overrides so they are environment specific

Reply:
Yes, just marked PR with WIP label.